packages/client/src/gen/coordinator/index.ts

```diff
@@ -911,6 +911,91 @@ export interface CallParticipantResponse {
    */
   user_session_id: string;
 }
+/**
+ *
+ * @export
+ * @interface CallRTMPBroadcast
+ */
+export interface CallRTMPBroadcast {
+  /**
+   *
+   * @type {string}
+   * @memberof CallRTMPBroadcast
+   */
+  CallID: string;
```

Comment:
This case is a bit weird, open api models usually follow snake case, could this be a mistake?